### PR TITLE
Added RSS feed to the homepage

### PR DESCRIPTION
**Pull request by McPringle**
- added IDs to the events on the website as an anchor
- implemented generation of static RSS feed.xml via gulp on build
- JavaScript could be cleaned up (hash function in `gulpfile.js` and `eventlist.js` is duplicate code)
Thanks to @codestoe for help and for @madmas for your idea… ;-)

---
This change is [Reviewable](https://reviewable.io/reviews/hackergarten/hackergarten.github.io/86)

### CODE REVIEW

**Comment by McPringle** [comment]
Looks great! Should be merged *immediately*! ;-)

---

Review status: 0 of 8 files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/hackergarten/hackergarten.github.io/86#-:-KM5eEJ6lehMoO5wUK2B:92890475)*

File: .travis.yml [hunk 1]
```diff
@@ -8,5 +8,13 @@ before_script:
...
 after_success: ./deploy.sh
 
+notifications:
+  webhooks:
+    urls:
+      - https://webhooks.gitter.im/e/8159b56253161aad725a
+    on_success: change  # options: [always|never|change] default: always
+    on_failure: always  # options: [always|never|change] default: always
+    on_start: never     # options: [always|never|change] default: always
+
```
Comment (madmas) [inline .travis.yml]: Cool idea! I like 👍

**Comment by madmas** [comment]
Hm, if I see the way "title" is used after this change correctly, venue would be a more suitable name for the attribute? what do you think?

**Comment by madmas** [comment]
Okay, I created #88 to track the naming issue, thanks a lot for the RSS feed effort, we should see it live soonish :)

**Pull request merged**